README.md

```diff
@@ -35,6 +35,48 @@ The following settings must be configured before running the app:
   Sets the hostname required by webpack for building the frontend. Should likely be whatever you set
   the host to in your /etc/hosts or the hostname that you're accessing it from. Likely `od.odl.local`.
 
+#### Keycloak configuration
+
+- `SOCIAL_AUTH_OL_OIDC_OIDC_ENDPOINT`
+
+  The base URI for OpenID Connect discovery, https://<OIDC_ENDPOINT>/ without .well-known/openid-configuration.
+
+- `OIDC_ENDPOINT`
+
+  The base URI for OpenID Connect discovery, https://<OIDC_ENDPOINT>/ without .well-known/openid-configuration.
+
+- `SOCIAL_AUTH_OL_OIDC_KEY`
+
+  The client ID provided by the OpenID Connect provider.
+
+- `SOCIAL_AUTH_OL_OIDC_SECRET`
+
+  The client secret provided by the OpenID Connect provider.
+
+- `AUTHORIZATION_URL`
+
+  Provider endpoint where the user is asked to authenticate.
+
+- `ACCESS_TOKEN_URL`
+
+  Provider endpoint where client exchanges the authorization code for tokens.
+
+- `USERINFO_URL`
+
+  Provder endpoint where client sends requests for identity claims.
+
+- `FEATURE_KEYCLOAK_ENABLED`
```

Comment:
Are all these env vars actually used? Most of them (e.g., USERINFO_URL, FEATURE_KEYCLOAK_ENABLED, but also others) I don't see usages of in the repo. (Are they for social django?)

Reply:
Most of these env vars are needed by social django.  Lots of hidden magic unfortunately.